metaflow/client/core.py

```diff
@@ -1837,6 +1974,60 @@ def environment_info(self) -> Optional[Dict[str, Any]]:
         for t in self:
             return t.environment_info
 
+    @property
+    def parent_steps(self) -> Iterator["Step"]:
```

Comment:
this is still duplicated?

Reply:
Wait, I didn;t push this change. Let me do that.